docs/intro/ff-ui/toolbar.md

```diff
@@ -0,0 +1,234 @@
+
+![command-palette.avif](imgs/command-palette.avif)
+
+## 5. Canvas size
+
+We allow you to visualize and design your app for various mobile, tablet, and desktop devices. The [canvas](#) size is defined as width (in pixels) x height (in pixels). This size is determined according to the device selected. To set a custom canvas size, click on the Canvas Size and enter the width and height (in pixels) you want.
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'
+}}>
+    <iframe 
+        src="https://demo.arcade.software/6TBNjYNfVDmV4NawxA4f?embed&show_copy_link=true"
+        title="Sharing a Project with a User"
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+## 6. Project history
+
+Versions
+
+Versions enable you to save specific named states of your project. This function is particularly useful for recovery purposes. If you accidentally delete a page or component or change settings, you can easily revert to a saved version.
+
+Snapshots
+
+Snapshots are automatically saved as you build.
+
+:::info
+
+Users on the *Free* and *Standard* plans can access only one day of snapshots.
+
+:::
+
+### 6.1 Saving a version
+
+Saving the latest changes will store the current state of your app in the version control. In the future, if anything goes wrong, you can restore your app to the currently saved state.
+
+Ideally, you should save the version whenever you accomplish something important, for example, after finishing a page, adding configuration, etc.
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'
+}}>
+    <iframe 
+        src="https://demo.arcade.software/d1rO0YLeMsrIt3Alb70E?embed&show_copy_link=true"
+        title="Sharing a Project with a User"
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+### 6.2 Restoring a version
+
+Restoring the previous version will preserve the current version, then load the changes from the version you're restoring. Before restoration, you may want to view the changes in the previous version. To do this, select the Peek option, which opens the previous version in a new tab.
```

Comment:
When mentioning FlutterFlow terms like "Peek option", you can add a bold styling, to **Peek** option so it stands out. Specially when you are mentioning it for the first time.